assets/tests/hooks/useDetour.test.ts

```diff
@@ -422,4 +422,49 @@ describe("useDetour", () => {
 
     await waitFor(() => expect(result.current.state).toBe(DetourState.Edit))
   })
+
+  describe("when `state` is `Finished`, controls are locked out", () => {
```

Comment:
**nitpick (non-blocking):** since these tests are all pretty similar maybe you could rewrite them with `test.each`? Not a big deal though because it's only a few of them and there isn't that much repeated code.

Reply:
Good point! Normally I'd agree, but there's a few refactors in the works, so I'll keep that in mind when working on them, but for now I think I'm going to leave it as is.

Reply:
Totally fair, I marked it non-blocking for a reason. 🙂